frame/dutch-auction/src/lib.rs

```diff
@@ -370,19 +470,101 @@ pub mod pallet {
 								real_quote_amount,
 							)
 							.expect("we forced locks beforehand");
-
 							if real_quote_amount < quote_amount {
 								T::MultiCurrency::unreserve(
 									order.pair.quote,
 									&take.from_to,
 									quote_amount - real_quote_amount,
 								);
+
+								amount_received += real_quote_amount;
 							}
 						}
 					}
 
 					if order.take.amount == T::Balance::zero() {
 						<SellOrders<T>>::remove(order_id);
+						if let Some((parachain_id, xcm_order_id)) =
+							LocalOrderIdToRemote::<T>::take(&order_id)
+						{
+							let received_amount =
+								T::MultiCurrency::free_balance(order.pair.quote, seller);
```

Comment:
need to think about. if account for selling will be only for selling, than all sold is on balance, and should be transferred. so really  need either to create subaccount from seller account and order id. or track sold amount in additional field. I think better do additional field. and retain one account of seller. 

so we need to know how much was sold to callback caller about how much of transfer was because of sold out.


but as of now there seems bug.